README.md

```diff
@@ -114,9 +122,11 @@ journalctl -fu docker.service # Journalctl/Systemd
 
 **VM (Docker-side) Plugin logs**
 
-* Log location: `/var/log/docker-volume-vsphere.log`
-* Config file location: `/etc/docker-volume-vsphere.conf`.
- * This JSON-formatted file controls logs retention, size for rotation
+* Log location (Linux): `/var/log/docker-volume-vsphere.log`
+* Log location (Windows): `C:\Windows\System32\config\systemprofile\AppData\Local\docker-volume-vsphere\logs\docker-volume-vsphere.log`
```

Comment:
This location is built upon the [`LOCALAPPDATA`](https://github.com/vmware/docker-volume-vsphere/blob/300327abfd03c94aafc226ca222904fb0562bc4e/client_plugin/utils/config/config_windows.go#L26) environment variable which returns `C:\Windows\System32\config\systemprofile\AppData\Local\` for Windows services, and hence that path.